packages/account/src/Sections/Security/ConnectedApps/connected-apps.tsx

```diff
@@ -86,78 +59,55 @@ const ConnectedApps = () => {
     }, []);
 
     const handleRevokeAccess = React.useCallback(() => {
-        setModalVisibility(false);
+        setIsModalOpen(false);
         revokeConnectedApp(selected_app_id);
     }, [revokeConnectedApp, selected_app_id]);
 
-    return (
-        <section
-            className={classNames('connected-apps__wrapper', {
-                'connected-apps__wrapper--dashboard': is_appstore,
-            })}
-        >
-            <Text color='prominent' weight='bold' as='p' className='connected-apps__title'>
-                {localize('Authorised applications')}
-            </Text>
-            {is_error && <ErrorComponent />}
-            <div
-                className={classNames('connected-apps__container', 'connected-apps__wrapper', {
-                    'connected-apps__wrapper--dashboard': is_appstore,
-                })}
-            >
-                {is_loading ? (
-                    <Loading is_fullscreen={false} />
+    return is_loading ? (
```

Comment:
`if(is_loading) return <Loading is_fullscreen={false} />` 
is a bit more readable, you can ignore this as well 🙏

Reply:
in other files as well, ternary operator is used for `is_loading`. so i think we can go with this.